spec/features/post_show_spec.rb

```diff
@@ -0,0 +1,52 @@
+require 'rails_helper'
+RSpec.describe 'posts#show', type: :feature do
+  before(:each) do
+    @user = User.create(
+      name: 'John',
+      photo: 'john.jpg',
+      bio: 'Web Developer',
+      posts_counter: 0
+    )
+    @posts = [
+      @post1 = Post.create(author: @user, title: 'User Post 1', text: 'This is User Post 1', likes_counter: 0,
+                           comments_counter: 0),
+      @post2 = Post.create(author: @user, title: 'User Post 2', text: 'This is User Post 2', likes_counter: 0,
+                           comments_counter: 0),
+      @post3 = Post.create(author: @user, title: 'User Post 3', text: 'This is User Post 3', likes_counter: 0,
+                           comments_counter: 0)
+    ]
+
+    # visit user_posts_url(user_id: @user.id)
```

Comment:
- [ ] Great job working on this milestone. In order to keep out codebase clean and professional, we should always delete all commented lines of code.